src/core/DominoNumber.qml

```diff
@@ -78,6 +84,25 @@ Item {
      */
     property GCSfx audioEffects
 
+    function getText() {
```

Comment:
you can directly have a "readonly property var romans: ["", "I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX"] and have below text:
text: mode == "number" ? item.value : romans[item.value]

I'll change this directly during the merge, no need to update this PR